src/flavors/__tests__/fixtures.ts

```diff
@@ -1,22 +1,6 @@
-import { Constructor } from '../../interfaces';
-
-import { BasicFlavor, OpenAPIFlavor } from '..';
-
-export function createFixtures(
-    /* It is rather difficult to write `createFixtures()` as a generic function
```

Comment:
All of the type signature challenges here go away now that flavors are not generic.